__main__ - optimizer_entrypoint(line:879) - !!! Skip optimizer !!! root_node_rps of all callgraph in all region is 0
__main__ - optimizer_entrypoint(line:879) - !!! Skip optimizer !!! root_node_rps of all callgraph in all region is 0
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 18:18:57 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 953, in optimizer_entrypoint
    weight = flow / total
ZeroDivisionError: division by zero
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 18:18:58 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 953, in optimizer_entrypoint
    weight = flow / total
ZeroDivisionError: division by zero
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 18:18:59 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 953, in optimizer_entrypoint
    weight = flow / total
ZeroDivisionError: division by zero
__main__ - optimizer_entrypoint(line:891) - !!! ERROR !!! Total demand(803) > total capcity(800)
__main__ - optimizer_entrypoint(line:892) - max_capacity_per_service[a][us-west-1]: 200
__main__ - optimizer_entrypoint(line:894) - new_capacity,a, 200 -> 210
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 18:19:58 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 953, in optimizer_entrypoint
    weight = flow / total
ZeroDivisionError: division by zero
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 18:19:59 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 953, in optimizer_entrypoint
    weight = flow / total
ZeroDivisionError: division by zero
